pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,355 @@
+                    <button type="button" class="btn btn-primary text-nowrap" data-toggle="modal" data-target="#copyAssessmentGroupsModal">
+                      <i class="fas fa-copy"></i> Copy
+                    </button>
+                    <div class="mt-2">
+                      Copy the group setting from another assessment.
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="table-responsive">
+          <table id="usersTable" class="table table-sm table-hover tablesorter">
+            <thead>
+              <tr>
+                <th>GID</th>
+                <th>Group Name</th>
+                <th>#Members</th>
+                <th class="text-center">Group Members (UIDs)</th>
+                <th class="text-center"></th>
+                <th class="text-center"></th>
+              </tr>
+            </thead>
+            <tbody>
+              <% groups_rows.forEach(function(row, iRow) { %>
+              <tr>
+                <td><%= row.gid %></td>
+                <td><%= row.groupname %></td>
+                <td class="text-center"><%= row.num %></td>
+                <td class="text-center"><%= row.uids %></td>
+                <td class="text-center">
+                  <div class="dropdown">
+                    <button type="button" class="btn btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      Action <span class="caret"></span>
+                    </button>
+                    <div id="row<%= iRow %>PopoverAdd"
+                      tabindex="0" data-toggle="popover" data-trigger="manual" data-container="body"
+                      data-html="true" data-placement="auto" title="Add Members"
+                      data-content="
+                                    <form name=&quot;add-member-form&quot; method=&quot;POST&quot;>
+                                      UIDs:
+                                      <input type=&quot;text&quot; class=&quot;form-control&quot; placeholder=&quot;A@ex.com, B@ex.com&quot; name=&quot;addmemberuids&quot;>
+                                      <br>
+                                      <input type=&quot;hidden&quot; name=&quot;__action&quot; value=&quot;addmember&quot;>
+                                      <input type=&quot;hidden&quot; name=&quot;__csrf_token&quot; value=&quot;<%= __csrf_token %>&quot;>
+                                      <input type=&quot;hidden&quot; name=&quot;gid&quot; value=&quot;<%= row.gid %>&quot;>
+                                      <button type=&quot;button&quot; class=&quot;btn btn-secondary&quot; onclick=&quot;$('#row<%= iRow %>PopoverAdd').popover('hide')&quot;>
+                                        Cancel
+                                      </button>
+                                      <button type=&quot;submit&quot; class=&quot;btn btn-primary&quot;>Add</button>
+                                    </form>
```

Comment:
Can we split off these popups into different files that we then do an EJS include on?  Something like https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/administratorOverview/administratorOverview.ejs#L81 as on the admin page perhaps.